frontends/web/src/locales/en/app.json

```diff
@@ -64,7 +64,7 @@
         "placeholder": "Recovery password",
         "showLabel": "recovery password"
       },
-      "success": "Successfully verified backup: {{name}}",
+      "success": "Successfully verified backup:",
```

Comment:
reminder: you can't do this here - needs to be via locize (then `make locize-pull`) (or now that you did it here, do the same again in locize).

otherwise the net `make locize-pull` will revert this.

Reply:
Right, thanks! Changed in locize